tests/lax_vmap_test.py

```diff
@@ -220,7 +220,6 @@ def testBitcastElementType(self, shape, from_dtype, to_dtype, bdims,):
       for bdims in all_bdims(min_shape, operand_shape, max_shape)))
   def testClamp(self, min_shape, operand_shape, max_shape, dtype, bdims):
     rng = jtu.rand_default(self.rng())
-    raise SkipTest("batching rule for clamp not implemented")  # TODO(mattj)
```

Comment:
Oh man, high five to my past self for these tests being here :D

Reply:
Yeah, great tests :)